apps/web/pages/api/apps/installed.ts

```diff
@@ -14,24 +14,24 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
     const userId = req.session.user.id;
     try {
-      const installedApp = await prisma.credential.findMany({
+      const credentials = await prisma.credential.findMany({
         where: {
           type: appCredentialType as string,
           userId: userId,
         },
       });
```

Comment:
This is a security risk. We're exposing private credential keys unnecessarily to the frontend.